docs/pages/reference/cli.mdx

```diff
@@ -67,6 +71,29 @@ clusters.
 | `--app-uri` | none | none | Internal address of the application to proxy |
 | `--app-public-addr` | none | none | Public address fo the application to proxy |
 
+#### teleport start --roles
+
+The `--roles` flag when used with `teleport --start` instructs Teleport on which specific Teleport services to start. Below is a more cohesive table of roles and their associated services that `teleport start` supports:
+
+| Service | Role Name | Description |
+| - | - | - |
+| [Node](../architecture/nodes.mdx) | `node` | Runs a daemon on a host that allows SSH connections from authenticated clients. |
+| [Auth](../architecture/authentication.mdx) | `auth` | Authenticates hosts and users who want access to Teleport-managed resources or information about a cluster. |
+| [Proxy](../architecture/proxy.mdx) | `proxy` | The gateway that clients use to connect to the Auth Service or resources managed by Teleport. |
+| [App](../application-access/introduction.mdx) | `app` | Runs a daemon on a host that provides access to applications using an SSH reverse tunnel. |
+| [Kube](../kubernetes-access/introduction.mdx) | `kube` | The Teleport daemon will run the Kubernetes Service. |
+| [DB](../database-access/reference.mdx) | `db` | The Teleport daemon will run the Database Service. |
+| [Trusted Cluster](../management/admin/trustedclusters.mdx) | `trusted_cluster` | The Teleport daemon will support a leaf cluster used to connect to another Teleport cluster. |
```

Comment:
Is `trusted_cluster` a role now? I don't see it in the master branch lib/config/configuration.go:validateRoles func.